hikari/impl/stateful_cache.py

```diff
@@ -125,7 +130,9 @@ def __init__(self, app: traits.RESTAware, intents: intents_.Intents) -> None:
         # found attached to cached presence activities.
         self._unknown_custom_emoji_entries = collections.FreezableDict()
         self._user_entries = collections.FreezableDict()
+        self._message_entries = collections.LimitedCapacityCacheMap(limit=max_messages)
         self._intents = intents
+        self.max_messages = max_messages
```

Comment:
We likely don't want to save the max messages as a public attribute like this as this'd mislead some people to think that modifying it will modify the amount of messages this cache will store